modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/integration.dm

```diff
@@ -6,9 +6,11 @@
 
 /datum/reagent/blood/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume, show_message=TRUE, touch_protection=0)
 	var/datum/antagonist/bloodsucker/bloodsuckerdatum = IS_BLOODSUCKER(exposed_mob)
-	if(!bloodsuckerdatum)
+	if(!bloodsuckerdatum || !/datum/bloodsucker_clan)
+		return ..()
+	if(/datum/bloodsucker_clan != "Venture Clan" && bloodsuckerdatum.bloodsucker_blood_volume >= BLOOD_VOLUME_NORMAL && reac_volume >= 0)
```

Comment:
This won't work, you can't compare a type to a string like this, and this isn't checking the clan datum of the bloodsucker antag